core/src/tests/test_fileindex_list.cc

```diff
@@ -118,8 +118,9 @@ static std::string ToBsrStringBareos(const itBegin& t_begin, const itEnd& t_end)
   });
   auto maxId = *std::max_element(t_begin, t_end);
   auto buffer = std::string{};
+  auto One = 1;
```

Comment:
I think `firstindex = 1` would make more sense to understand what `1` stands for

Reply:
good catch